### PR TITLE
Fix bug where the wrong requirement is removed

### LINKED ISSUE #411: Removing a requirement from the info icon removes the last requirement not the selected one

### PR DESCRIPTION
**Pull request by miriam-rittenberg**
Fixes #411

### CODE REVIEW

File: src/components/Audit.vue [hunk 1]
```diff
@@ -420,7 +420,12 @@ export default {
       return `--percent: ${pfulfilled}%; --bar-color: ${pcolor}; --bg-color: #fff`;
     },
     deleteReq: function (req) {
-      const reqName = req['list-id'].substring(0, req['list-id'].indexOf('.reql'));
+      let reqName;
+      if (req['list-id'].includes('.reql')) {
+        reqName = req['list-id'].substring(0, req['list-id'].indexOf('.reql'));
```
Comment (georgiashay) [inline src/components/Audit.vue:425]: Are there any situations where it takes this branch?  I tested a bunch and it seemed like it never had the .reql

**Review by georgiashay — COMMENTED** [review]
Looks good to me, but we can possibly remove the first branch of the if statement unless there are situations where it is taken.

File: src/components/Audit.vue (hunk 1 above)
Comment (miriam-rittenberg) [reply src/components/Audit.vue:425]: I think you're right that it never has .reql

**Comment by georgiashay** [comment]
Looks good

**Pull request merged**